src/classic/clvm_tools/stages/stage_2/module.rs

```diff
@@ -493,59 +500,59 @@ fn build_macro_lookup_program(
             &mut macros.iter()
         );
         Ok(result_program)
-    };
+    }
 }
 
+#[allow(clippy::too_many_arguments)]
 fn add_one_function(
     allocator: &mut Allocator,
     args_root_node: &NodePath,
     macro_lookup_program: NodePtr,
     constants_symbol_table: &[(NodePtr, Vec<u8>)],
-    compiled_functions_: HashMap<Vec<u8>, NodePtr>,
+    mut compiled: CompileOutput,
```

Comment:
Could use a comment that `compiled` is an in/out parameter

Reply:
ya good point.

Reply:
So my rust style usually has by-value parameters _not_ be mutable, and I move into a mutable value inside the function.  This is a parameter by value that is allowed to be mutated by the body but the mutability is not visible to the caller.

In reviews, others have criticized my first instinct style so i've taken to the briefer, but in my opinion less clear style.

Reply:
I agree with this style. It's the same as C++